spec/mailers/conversation_reply_mailer_spec.rb

```diff
@@ -171,7 +171,7 @@
 
       it 'renders assignee name in the from address' do
         mail = described_class.email_reply(message)
-        expect(mail['from'].value).to eq "#{conversation.assignee.available_name} from #{smtp_email_channel.inbox.name} <#{smtp_email_channel.email}>"
+        expect(mail['from'].value).to eq "#{message.sender.name} from #{smtp_email_channel.inbox.name} <#{smtp_email_channel.email}>"
```

Comment:
@tejaswinichile Can you please add a case for the message sender name being unavailable as well as assigned agent name also unavailable?

Reply:
Yes.

Reply:
@pranavrajs 
1. We have a DB constraint that the name can't be blank, so saving an empty name for the sender is impossible - we can not test if the sender is present with a blank name.
2. We can have the empty sender for the message; [need to do something for this association] adding the specs to handle the empty sender.